debian/conf/nouveau.yaml

```diff
@@ -0,0 +1,28 @@
+maxIndexesOpen: 5000
+commitIntervalSeconds: 30
+idleSeconds: 60
+rootDir: /var/opt/nouveau/indexes
+
+logging:
+  level: INFO
+
+server:
+  applicationConnectors:
+    - type: http
+      bindHost: 127.0.0.1
+      port: 5987
+      useDateHeader: false
+  gzip:
+    includedMethods:
+      - GET
+      - POST
+  requestLog:
+    appenders:
+      - type: file
+        archive: true
+        archivedFileCount: 5
+        archivedLogFilenamePattern: /var/opt/nouveau/log/nouveau-%i.log
```

Comment:
Does this mean Dropwizard does its own log rotation?

Reply:
Jetty does, and nouveau exposes it.